### PR TITLE
Retrieve DrugBank info from json instead of html

### PR DESCRIPTION
**Pull request by AstrorEnales**
DrugBank provides json data on all released versions (https://go.drugbank.com/releases.json) which should be more stable to retrieve than parsing html.

### CODE REVIEW

**Comment by cthoyt** [comment]
@AstrorEnales this is great, thank you! Note I made a few changes to make the python code more idiomatic.

**Comment by cthoyt** [comment]
There are a few other fixes on the main branch I need to make first, then I will merge this.

**Comment by AstrorEnales** [comment]
Thank you for the quick response. I tested the changes and fixed it from min to max.

File: src/bioversions/sources/drugbank.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def get(self):
         """Get the latest DrugBank version number."""
         res = requests.get(URL)
         res.raise_for_status()
-        latest = min(res.json(), key=itemgetter("released_on"))
+        latest = max(res.json(), key=itemgetter("released_on"))
```
Comment (cthoyt) [inline src/bioversions/sources/drugbank.py:31]: thank you for the typo catch :) I am sick today and not so with it

**Comment by cthoyt** [comment]
thank you @AstrorEnales !

**Pull request merged**